### PR TITLE
Science support fixes

### PR DESCRIPTION
**Pull request by eheinrich**
Fixes:
- Optical element not populated when loading an observation request. This was because the dropdown values were case sensitive. I've just lowercased all optical element codes in the dropdowns, I think this should be ok but let me know what you think.
- Defocus not populated when loading observation request.
- Fill window calculation bug
- Allow eccentricity values >= 1
- Add filter by requestgroup name to front page
- Fix url for observations associated with a request

### CODE REVIEW

**Comment by jnation3406** [comment]
sorry I broke a test in one of my changes for ASAS last night, I fixed it in my latest pull request with priority

File: static/js/request_row.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ $(document).ready(function() {
   $('.pending-details').each(function() {
     let that = $(this);
     let requestId = $(this).data('request');
-    $.getJSON('/api/requests/' + requestId + '/?exclude_canceled=true', function(data) {
+    $.getJSON('/api/requests/' + requestId + '/observations/?exclude_canceled=true', function(data) {
```
Comment (jnation3406) [inline static/js/request_row.js]: I wonder how this got in here... this should have never worked with that URL

Comment (eheinrich) [reply static/js/request_row.js]: Yeah, I guess it probably just slipped by unnoticed :woman_shrugging:

**Comment by eheinrich** [comment]
Ok, I'll wait until the other branch is merged in to master, then I'll get the branch passing and merge it in

**Pull request merged**